File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        l1 = len(word1)
        l2 = len(word2)

        rslt = ""
        i = 0   # word1 index  
        j = 0   # word2 index

        while i < l1 and j < l2:
            rslt += word1[i]
            rslt += word2[j]
            i+=1
            j+=1
        
        while i < l1:
            rslt += word1[i]
            i+=1
        
        while j < l2:
            rslt += word2[j]
            j += 1
        
        return rslt

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.go
func mergeAlternately(word1 string, word2 string) string {
    var merged = ""

    var i, j = 0, 0
    var l1, l2 = len(word1), len(word2)

    for i < l1 && j < l2{
        merged = merged + string(word1[i])
        merged = merged + string(word2[j])
        i += 1
        j += 1
    }

    for i < l1{
        merged += string(word1[i])
        i += 1
    }

    for j < l2 {
        merged += string(word2[j])
        j += 1
    }

    return merged
}